packages/hooks-react/src/usePlanByEpg.ts

```diff
@@ -72,19 +70,17 @@ const usePlanByEpg = ({
   });
 };
 
-// Theme configuration for the Planby EPG with only the colors set that are used
-// Fixed values are used because the default highlightColor and backgroundColor are only available in SCSS
-export const makeTheme = (primaryColor?: string | null, backgroundColor?: string | null) => ({
+export const makeTheme = () => ({
   primary: {
-    600: backgroundColor || '#141523',
-    900: backgroundColor || '#141523',
+    600: '#141523',
```

Comment:
Don't we want a neutral (grayscale) colour by default here?

Reply:
Yes, I still had doubts about that, but I think I found a way! I added a CSS variable for the EPG background color which is set when using a darker background color.